app/controllers/schedule_controller.rb

```diff
@@ -2,35 +2,44 @@
 class ScheduleController < ApplicationController
   def index
     @user = current_user
-    render 'admin_index' if @user.coach?
-    # create schedule if one doesn't exist
-    @schedule = if @user.schedule
-                  @user.schedule
-                else
-                  @user.schedule = Schedule.create
-                end
+    if @user.coach?
+      render 'admin_index' if @user.coach?
```

Comment:
Is this extra `if` redundant?

Reply:
Yes it is... I was refactoring a few times and I missed this - good catch!